delivery_auto_refresh/data/ir_config_parameter.xml

```diff
@@ -2,6 +2,10 @@
 <!-- Copyright 2018 Tecnativa - Pedro M. Baeza
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo noupdate="1">
+    <record id="set_default_carrier" model="ir.config_parameter">
```

Comment:
this param should be set from res.config.settings IMO but as other params here are not.... I understand if you don't do it ;)